cloud/image_classification/README.md

```diff
@@ -71,7 +71,7 @@ cd ../../loadgen; CFLAGS="-std=c++14" python setup.py develop; cd ../cloud/image
 
 python setup.py develop
 ```
-
+Install the loadgen module under dist/ with pip install. Add -I to override any existing loadgen module..
```

Comment:
Perhaps add a link to the loadgen README?
https://github.com/mlperf/inference/blob/master/loadgen/README.md

Reply:
Good idea.